gamification/process/update.py

```diff
@@ -135,6 +136,33 @@ def get_participant_objects():
     return participants
 
 
+def get_meta_review_participant_objects():
+    return meta_review_participants.objects.all()
+
+
+def update_participants_data_with_meta_review(participant):
+    """
+    Update participants based on meta-review
+
+    This method updates every participant based on the meta-review
+    received or given. If the meta-review is completed then it calls
+    'Completed Meta-Review' activity.
+    """
+    pos_in_meta_review = participant.pos_in
+    pos_out_meta_review = participant.pos_out
+    neg_in_meta_review = participant.neg_in
+    neg_out_meta_review = participant.neg_out
+    if pos_in_meta_review > 0 or pos_out_meta_review > 0 or \
+       neg_in_meta_review > 0 or neg_out_meta_review > 0:
+        """
+        Add a meta-review activity to the participant
+        """
+        activity = Activity.object.create(
```

Comment:
Try to use [add_activity](https://github.com/coala/community/blob/da5ada362c8fc3ae2cb179739a4babbfad1e7d1f/gamification/models.py#L135) method.